src/context/color-mode.context.jsx

```diff
@@ -26,6 +26,12 @@ export function ColorModeProvider({ children }) {
   const theme = React.useMemo(
     () => createTheme({
       palette: {
+        primary: {
+          main: '#f07e00',
+        },
+        secondary: {
+          main: '#ffb74d',
+        },
```

Comment:
Die Farben sind auch MW spezifisch und gehören hier nicht rein.
Bitte benenne den Color-Mode-Provider zu Theme-Provider um (auch den Kontext) und erstelle einen zweiten State für die ganze Palette. Dann übergibst du das `setPalette` im Provider (den bisherigen `colorMode` spreadest du rein und ergänzt in mit dem `setPalette`).  

Im UseMemo verwendest du dann den palette-State und überschreibst den mode mit dem bisherigen Code.

Danach kannst du im shopify-bq-client mit useContext(ThemeContext) die `setPalette` Funktion holen und die Farben dort setzen.